src/main/java/com/online/cinema/util/HibernateUtil.java

```diff
@@ -14,9 +13,7 @@ private HibernateUtil() {
 
     private static SessionFactory initSessionFactory() {
         try {
-            Configuration configuration = new Configuration();
-            configuration.addAnnotatedClass(Movie.class);
-            return configuration.buildSessionFactory();
+            return new Configuration().configure().buildSessionFactory();
         } catch (Exception e) {
             throw new DataProcessingException("Error creating session factory", e);
```

Comment:
```suggestion
            throw new RuntimeException("Error creating session factory", e);
```
this exception has a different nature